internal/serving/testing/minimatch.go

```diff
@@ -53,31 +53,28 @@ func (mm *MiniMatchServer) Stop() {
 }
 
 // MustMiniMatch requires Mini Match to be created successfully.
-func MustMiniMatch(params []*serving.ServerParams) (*MiniMatchServer, func()) {
-	mm, closer, err := NewMiniMatch(params)
+func MustMiniMatch(params []*serving.ServerParams) *MiniMatchServer {
+	mm, err := NewMiniMatch(params)
 	if err != nil {
 		panic(err)
 	}
-	return mm, closer
+	return mm
 }
 
 // NewMiniMatch creates and starts an OpenMatchServer context for testing.
-func NewMiniMatch(params []*serving.ServerParams) (*MiniMatchServer, func(), error) {
+func NewMiniMatch(params []*serving.ServerParams) (*MiniMatchServer, error) {
```

Comment:
Why is this being removed? This leaks a grpc server, redis, etc..

Reply:
Correct me if I am wrong. I thought calling `defer mm.Stop()` in https://github.com/GoogleCloudPlatform/open-match/blob/master/internal/serving/testing/minimatch_test.go#L33 is essentially the same as `defer closer()` https://github.com/GoogleCloudPlatform/open-match/blob/master/internal/serving/testing/minimatch_test.go#L41.

Reply:
That's the only test that calls mm.Stop() directly the others use closer(). This test calls mm.Stop() to verify that the server is shutdown.